mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -374,8 +404,21 @@ class ALS(@Since("1.4.0") override val uid: String) extends Estimator[ALSModel]
   @Since("1.3.0")
   def setSeed(value: Long): this.type = set(seed, value)
 
+  /** @group expertSetParam */
+  @Since("2.0.0")
+  def setIntermediateRDDStorageLevel(value: String): this.type = {
+    set(intermediateRDDStorageLevel, value)
```

Comment:
@MLnick Don't we need to verify the value to be one of the StorageLevel here? Correct me if I am wrong.